package.json

```diff
@@ -62,6 +62,9 @@
     "@types/keyv": "3.1.4",
     "@types/react": "17.0.24",
     "@types/react-dom": "17.0.16",
+    "@polkadot/api": "10.9.1",
+    "@polkadot/types": "10.9.1",
+    "@substrate/txwrapper-core": "7.0.1",
```

Comment:
`@substrate/txwrapper-substrate` needs to be `7.0.1` as well

Reply:
Also need to confirm that packages like `@bitgo-beta/sdk-doin-dot` when installed don't include the unintended versions.